### PR TITLE
Add support for creating a Checkout Session via the CLI

### LINKED ISSUE #151: Support checkout webhooks
### Problem
Testing checkout webhook
checkout.session.completed
I cant even find docs what will object checkout.session.completed look like in webhook and cant test with cli and testing on real domains or tunnels is a pain

### Feature
trigger checkout.session.completed

### PR DESCRIPTION
**Pull request by joek-stripe**
### Reviewers
r? @ob-stripe 
cc @stripe/dev-platform

 ### Summary
Added `stripe trigger checkout.session.created` which will exercise the new checkout session flow from the CLI

[dx-4745] closes #151

### CODE REVIEW

File: pkg/cmd/trigger.go [hunk 1]
```diff
@@ -31,6 +31,7 @@ func newTriggerCmd() *triggerCmd {
 			"charge.failed",
 			"charge.refunded",
 			"charge.succeeded",
+			"checkout.session.created",
```
Comment (ob-stripe) [inline pkg/cmd/trigger.go]: The event type is `checkout.session.completed`, not `.created`.

File: pkg/requests/examples.go [hunk 2]
```diff
@@ -140,6 +142,63 @@ func (ex *Examples) ChargeSucceeded() error {
 	return err
 }
 
+// SessionCreated creates a checkout session
+// https://stripe.com/docs/api/checkout/sessions/create?lang=curl
+func (ex *Examples) SessionCreated() error {
```
Comment (ob-stripe) [inline pkg/requests/examples.go]: Can you rename the method to `CheckoutSessionCompleted` and update the comment too?

**Comment by joek-stripe** [comment]
ptal @ob-stripe

File: pkg/requests/examples.go [hunk 3]
```diff
@@ -140,6 +142,63 @@ func (ex *Examples) ChargeSucceeded() error {
 	return err
 }
 
+// CheckoutSessionCompleted creates a checkout session
```
Comment (ob-stripe) [inline pkg/requests/examples.go]:
```suggestion
// CheckoutSessionCompleted creates and completes a Checkout session.
```

File: pkg/requests/examples_test.go [hunk 4]
```diff
@@ -382,3 +383,63 @@ func TestResendEventDoesNotErrorWithValidEventID(t *testing.T) {
 	t.Log(err)
 	require.Nil(t, err)
 }
+
+func TestSessionCompleted(t *testing.T) {
```
Comment (ob-stripe) [inline pkg/requests/examples_test.go]:
```suggestion
func TestCheckoutSessionCompleted(t *testing.T) {
```

**Pull request merged**